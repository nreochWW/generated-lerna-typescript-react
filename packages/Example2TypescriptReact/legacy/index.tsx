import * as React from "react";
import * as ReactDOM from "react-dom";

import Example2TypescriptReact from "../src/index";

ReactDOM.render(
  <Example2TypescriptReact compiler="TypeScript" framework="React" />,
  document.getElementById("Example2TypescriptReact")
);
