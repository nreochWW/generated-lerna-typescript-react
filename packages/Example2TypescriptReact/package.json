{
  "name": "example-2-typescript-react",
  "description": "Another Example component using lerna-typescript-react geneator",
  "version": "0.0.3",
  "main": "lib/index.js",
  "license": "MIT",
  "scripts": {
    "tsc": "tsc",
    "legacy": "webpack --config legacy/webpack.config.js"
  },
  "dependencies": {
    "example-typescript-react": "^0.0.3"
  }
}
