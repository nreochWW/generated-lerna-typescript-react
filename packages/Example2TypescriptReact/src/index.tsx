import * as React from "react";
import styled from "styled-components";

export interface Example2TypescriptReactProps {
  compiler: string;
  framework: string;
}

const Wrapper = styled.div`
  border: 1px solid blue;
  padding: 10px;
`;

const Example2TypescriptReact = (props: Example2TypescriptReactProps) => (
  <Wrapper>
    <h1>
      Another Example: {props.compiler} and {props.framework}!
    </h1>
  </Wrapper>
);

export default Example2TypescriptReact;
