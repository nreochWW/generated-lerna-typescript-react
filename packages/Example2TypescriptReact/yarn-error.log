Arguments: 
  /Users/nigel.reoch/.nvm/versions/node/v10.16.3/bin/node /Users/nigel.reoch/.nvm/versions/node/v10.16.3/bin/yarn install --mutex network:42424 --non-interactive

PATH: 
  /Users/nigel.reoch/nigel-development/generators/generated-lerna-typescript-react/packages/Example2TypescriptReact/node_modules/.bin:/Users/nigel.reoch/nigel-development/generators/generated-lerna-typescript-react/packages/node_modules/.bin:/Users/nigel.reoch/nigel-development/generators/generated-lerna-typescript-react/node_modules/.bin:/Users/nigel.reoch/nigel-development/generators/node_modules/.bin:/Users/nigel.reoch/nigel-development/node_modules/.bin:/Users/nigel.reoch/node_modules/.bin:/Users/node_modules/.bin:/node_modules/.bin:/Users/nigel.reoch/.nvm/versions/node/v10.16.3/bin:/var/folders/7q/fl711l2d3j15wmmcyd9wkz540000gp/T/yarn--1568214403234-0.001479574861072308:/Users/nigel.reoch/nigel-development/generators/generated-lerna-typescript-react/node_modules/.bin:/Users/nigel.reoch/.config/yarn/link/node_modules/.bin:/Users/nigel.reoch/.nvm/versions/node/v10.16.3/libexec/lib/node_modules/npm/bin/node-gyp-bin:/Users/nigel.reoch/.nvm/versions/node/v10.16.3/lib/node_modules/npm/bin/node-gyp-bin:/Users/nigel.reoch/.nvm/versions/node/v10.16.3/bin/node_modules/npm/bin/node-gyp-bin:/Users/nigel.reoch/.nvm/versions/node/v10.16.3/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin

Yarn version: 
  1.17.3

Node version: 
  10.16.3

Platform: 
  darwin x64

Trace: 
  Error: https://registry.yarnpkg.com/example-typescript-react: Not found
      at Request.params.callback [as _callback] (/Users/nigel.reoch/.nvm/versions/node/v10.16.3/lib/node_modules/yarn/lib/cli.js:66830:18)
      at Request.self.callback (/Users/nigel.reoch/.nvm/versions/node/v10.16.3/lib/node_modules/yarn/lib/cli.js:140464:22)
      at Request.emit (events.js:198:13)
      at Request.<anonymous> (/Users/nigel.reoch/.nvm/versions/node/v10.16.3/lib/node_modules/yarn/lib/cli.js:141436:10)
      at Request.emit (events.js:198:13)
      at IncomingMessage.<anonymous> (/Users/nigel.reoch/.nvm/versions/node/v10.16.3/lib/node_modules/yarn/lib/cli.js:141358:12)
      at Object.onceWrapper (events.js:286:20)
      at IncomingMessage.emit (events.js:203:15)
      at endReadableNT (_stream_readable.js:1145:12)
      at process._tickCallback (internal/process/next_tick.js:63:19)

npm manifest: 
  {
  	"name": "example-2-typescript-react",
  	"description": "Another Example component using lerna-typescript-react geneator",
  	"version": "0.0.0",
  	"main": "lib/index.js",
  	"license": "MIT",
  	"dependencies": {
  		"example-typescript-react": "^0.1.0"
  	}
  }

yarn manifest: 
  No manifest

Lockfile: 
  No lockfile
