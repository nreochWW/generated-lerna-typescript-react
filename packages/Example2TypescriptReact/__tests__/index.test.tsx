import * as React from "react";
import { shallow } from "enzyme";
import Example2TypescriptReact from "../src/index";

describe("Example2TypescriptReact package tests", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(
      <Example2TypescriptReact compiler="TypeScript" framework="React" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
