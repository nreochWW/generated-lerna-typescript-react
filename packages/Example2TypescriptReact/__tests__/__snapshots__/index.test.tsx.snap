// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example2TypescriptReact package tests matches snapshot 1`] = `
<styled.div>
  <h1>
    Another Example: 
    TypeScript
     and 
    React
    !
  </h1>
</styled.div>
`;
